### ISSUE TITLE
Sidebar hover not moving/working? 

### ISSUE THREAD

**Issue by mattpopovich** [cotes2020/jekyll-theme-chirpy#849]
## Checklist

- [x] I have read the [newlest tutorials](https://cotes2020.github.io/chirpy-demo/categories/tutorial/) and know the correct effect of the functional design.
- [x] There is no similar question on [existing issues](https://github.com/cotes2020/jekyll-theme-chirpy/issues?q=is%3Aissue) (including closed ones).
- [ ] I have tried to find the answer on [Jekyll Forum](https://talk.jekyllrb.com/) and [StackOverflow](https://stackoverflow.com/questions/tagged/jekyll).
- [x] My question is based on the latest code of the `master` branch.

## Description

Hi again cotes. For some reason, whenever I serve or build and deploy [my website](https://mattpopovich.com) (built [from chirpy-starter](https://github.com/mattpopovich/mattpopovich.com)), the sidebar tab indicator does not move upon mouse hover. 

I feel like I am deploying or building my site incorrectly or there are some files missing in chirpy-starter that I didn't copy, etc. 

### My build and deploy process
Using a M1 Mac running Monterey:
* Add `gem "jekyll-sass-converter", "~> 2.0"` to end of Gemfile. [See commit here](https://github.com/mattpopovich/mattpopovich.com/commit/fc008cbc9f33275855185ffdbdc6d5a7c1740d37) (thanks for the reminder @NichtsHsu)
* Enter the jekyll/jekyll:4.2.2 container
* `cd /path/to/mattpopovich.com`
* `git config --system --add safe.directory /path/to/mattpopovich.com`
* `bundle`
* `JEKYLL_ENV=production bundle exec jekyll b`
* rsync `_site` folder to my host

### Debugging
``` console
# bundle info --path jekyll-theme-chirpy 
/usr/gem/gems/jekyll-theme-chirpy-5.4.0
# ls /usr/gem/gems/jekyll-theme-chirpy-5.4.0/
LICENSE    README.md  _data      _includes  _layouts   _sass      assets
# ls /usr/gem/gems/jekyll-theme-chirpy-5.4.0/_includes/
assets-origin.html     embed                  head.html              mermaid.html           post-paginator.html    related-posts.html     toc.html
comments               favicons.html          js-selector.html       mode-toggle.html       post-sharing.html      search-loader.html     topbar.html
comments.html          footer.html            jsdelivr-combine.html  no-linenos.html        read-time.html         search-results.html    trending-tags.html
datetime.html          google-analytics.html  language-alias.html    post-nav.html          refactor-content.html  sidebar.html           update-list.html
# ls /usr/gem/gems/jekyll-theme-chirpy-5.4.0/assets/js/* 
data:
search.json  swcache.js

dist:
categories.min.js  commons.min.js     home.min.js        misc.min.js        page.min.js        post.min.js        pvreport.min.js

pwa:
app.js         sw.js          unregister.js
```

### Testing
Verified the sidebar hover not working on 
* M1 Mac, Chrome
* Intel Mac, Safari

So I think I have all the necessary files...? Thoughts? 

Thanks again for your hard work creating and maintaining this repo!

**Comment by mattpopovich** [comment]
If the problem is that [jekyll/jekyll](https://hub.docker.com/r/jekyll/jekyll/tags) uses Alpine which uses [musl libc](https://wiki.musl-libc.org/getting-started.html) which gives a `NotImplementedError` for `sass_embedded 1.57.1`, I could work on making a Dockerfile FROM Ubuntu based on [the GitHub Workflows](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/.github/workflows/ci.yml) that we could use instead of jekyll/jekyll.

**Comment by cotes2020** [comment]
Hey guys, there are two issues involved here:
1. The Gemfile regexp in 9f88426 didn't work;
2. The division formula of tab hover is not backward compatible to `jekyll-sass-convert 2.x`

So I made the fix in 808d374, it's been tested in `jekyll-sass-convert 2.x` and `jekyll-sass-convert 3.x`, hope this is a happy ending: `linux-musl` and Dart Sass 1.33.0+ compatible.

**Comment by mattpopovich** [comment]
I can confirm that the 808d374 fix to the `Gemfile` and `_sass/addon/commons.scss` fixes the sidebar hover issue for me. 

Thanks, @cotes2020!
